Content.Server/Drone/DroneSystem.cs

```diff
@@ -175,8 +176,12 @@ private bool NonDronesInRange(EntityUid uid, DroneComponent component)
             var xform = Comp<TransformComponent>(uid);
             foreach (var entity in _lookup.GetEntitiesInRange(xform.MapPosition, component.InteractionBlockRange))
             {
+                // Return true if the entity is/was controlled by a player and is not a drone or ghost.
                 if (HasComp<MindComponent>(entity) && !HasComp<DroneComponent>(entity) && !HasComp<GhostComponent>(entity))
```

Comment:
```suggestion
                if (HasComp<MobStateComponent>(entity) && !HasComp<DroneComponent>(entity) && !HasComp<GhostComponent>(entity))
```
Not all player controlled things should block a drone's actions (such as vending machines). I tested this change on my local repo

Reply:
read your laws